website/docs/sql/error_codes.md

```diff
@@ -0,0 +1,146 @@
+# Error Codes
+
+All messages emitted by Hyper are assigned five-character error codes
```

Comment:
This needs to be tied into the remaining documentation somehow. E.g., describe where to retrieve those error codes from